app/views/merchant/dashboard/index.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Merchant Dashboard</h1>
+<% if @merchant %>
+<h1><%= @merchant.name %></h1>
+
+<h3><%= @merchant.address %> <%= @merchant.city %> <%= @merchant.state %> <%= @merchant.zip %></h3>
+
+<ul>
+  <div id="order_info">
+  <% @merchant.orders.each do |order| %>
+    <li>Order: <%= link_to order.id, "/orders/#{order.id}"%></li>
+    <li>Quantity: <%= order.total_quantity %></li>
+    <li>Subtotal: <%= order.grandtotal %></li>
+    <li>Order Date: <%= order.created_at.to_s[0..9] %></li><br>
```

Comment:
Looks good Logan! Another way to format dates is with the `strftime` method. Just FYI - no changes needed.